### PR TITLE
feat(*): update build dockerfiles

### PR DESCRIPTION
**Pull request by judgeeeeee**
**What this PR does / why we need it**:

Add your description

update build 

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @your-reviewer

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by judgeeeeee** [comment]
docker build -t ghcr.io/kleveross/model-registry:v0.2.0-1-g7a635cc-dirty --label git-describe=v20201123-v0.2.0-1-g7a635cc-dirty  -f ./build/model-registry/Dockerfile .
docker build -t ghcr.io/kleveross/modeljob-operator:v0.2.0-1-g7a635cc-dirty --label git-describe=v20201123-v0.2.0-1-g7a635cc-dirty  -f ./build/modeljob-operator/Dockerfile .
# build extractor
docker build -t ghcr.io/kleveross/caffe-extract:v0.2.0-1-g7a635cc-dirty --label git-describe=v20201123-v0.2.0-1-g7a635cc-dirty  -f ./build/extract/caffe/Dockerfile .
docker build -t ghcr.io/kleveross/caffe2-extract:v0.2.0-1-g7a635cc-dirty --label git-describe=v20201123-v0.2.0-1-g7a635cc-dirty  -f ./build/extract/caffe2/Dockerfile .
docker build -t ghcr.io/kleveross/graphdef-extract:v0.2.0-1-g7a635cc-dirty --label git-describe=v20201123-v0.2.0-1-g7a635cc-dirty  -f ./build/extract/graphdef/Dockerfile .
docker build -t ghcr.io/kleveross/h5-extract:v0.2.0-1-g7a635cc-dirty --label git-describe=v20201123-v0.2.0-1-g7a635cc-dirty  -f ./build/extract/h5/Dockerfile .
docker build -t ghcr.io/kleveross/mxnetparams-extract:v0.2.0-1-g7a635cc-dirty --label git-describe=v20201123-v0.2.0-1-g7a635cc-dirty  -f ./build/extract/mxnetparams/Dockerfile .
docker build -t ghcr.io/kleveross/onnx-extract:v0.2.0-1-g7a635cc-dirty --label git-describe=v20201123-v0.2.0-1-g7a635cc-dirty  -f ./build/extract/onnx/Dockerfile .
docker build -t ghcr.io/kleveross/pmml-extract:v0.2.0-1-g7a635cc-dirty --label git-describe=v20201123-v0.2.0-1-g7a635cc-dirty  -f ./build/extract/pmml/Dockerfile .
docker build -t ghcr.io/kleveross/savedmodel-extract:v0.2.0-1-g7a635cc-dirty --label git-describe=v20201123-v0.2.0-1-g7a635cc-dirty  -f ./build/extract/savedmodel/Dockerfile .
docker build -t ghcr.io/kleveross/torchscript-extract:v0.2.0-1-g7a635cc-dirty --label git-describe=v20201123-v0.2.0-1-g7a635cc-dirty  -f ./build/extract/torchscript/Dockerfile .
# build convertor
docker build -t ghcr.io/kleveross/caffemodel_to_netdef:v0.2.0-1-g7a635cc-dirty --label git-describe=v20201123-v0.2.0-1-g7a635cc-dirty  -f ./build/convert/caffemodel_to_netdef/Dockerfile .
docker build -t ghcr.io/kleveross/h5_to_savedmodel:v0.2.0-1-g7a635cc-dirty --label git-describe=v20201123-v0.2.0-1-g7a635cc-dirty  -f ./build/convert/h5_to_savedmodel/Dockerfile .
docker build -t ghcr.io/kleveross/mxnetparams_to_onnx:v0.2.0-1-g7a635cc-dirty --label git-describe=v20201123-v0.2.0-1-g7a635cc-dirty  -f ./build/convert/mxnetparams_to_onnx/Dockerfile .
docker build -t ghcr.io/kleveross/mlserver:v0.2.0-1-g7a635cc-dirty --label git-describe=v20201123-v0.2.0-1-g7a635cc-dirty  -f ./build/serving/mlserver/Dockerfile .
docker build -t ghcr.io/kleveross/openscoring:v0.2.0-1-g7a635cc-dirty --label git-describe=v20201123-v0.2.0-1-g7a635cc-dirty  -f ./build/serving/openscoring/Dockerfile .
docker build -t ghcr.io/kleveross/tritonserver:v0.2.0-1-g7a635cc-dirty --label git-describe=v20201123-v0.2.0-1-g7a635cc-dirty  -f ./build/serving/tritonserver/Dockerfile .


docker push  ghcr.io/kleveross/model-registry:v0.2.0-1-g7a635cc-dirty
docker push  ghcr.io/kleveross/modeljob-operator:v0.2.0-1-g7a635cc-dirty
# push extractor
docker push  ghcr.io/kleveross/caffe-extract:v0.2.0-1-g7a635cc-dirty
docker push  ghcr.io/kleveross/caffe2-extract:v0.2.0-1-g7a635cc-dirty
docker push  ghcr.io/kleveross/graphdef-extract:v0.2.0-1-g7a635cc-dirty
docker push  ghcr.io/kleveross/h5-extract:v0.2.0-1-g7a635cc-dirty
docker push  ghcr.io/kleveross/mxnetparams-extract:v0.2.0-1-g7a635cc-dirty
docker push  ghcr.io/kleveross/onnx-extract:v0.2.0-1-g7a635cc-dirty
docker push  ghcr.io/kleveross/pmml-extract:v0.2.0-1-g7a635cc-dirty
docker push  ghcr.io/kleveross/savedmodel-extract:v0.2.0-1-g7a635cc-dirty
docker push  ghcr.io/kleveross/torchscript-extract:v0.2.0-1-g7a635cc-dirty
# push convertor
docker push  ghcr.io/kleveross/caffemodel_to_netdef:v0.2.0-1-g7a635cc-dirty
docker push  ghcr.io/kleveross/h5_to_savedmodel:v0.2.0-1-g7a635cc-dirty
docker push  ghcr.io/kleveross/mxnetparams_to_onnx:v0.2.0-1-g7a635cc-dirty
#push serving 
docker push  ghcr.io/kleveross/mlserver:v0.2.0-1-g7a635cc-dirty
docker push  ghcr.io/kleveross/openscoring:v0.2.0-1-g7a635cc-dirty
docker push  ghcr.io/kleveross/tritonserver:v0.2.0-1-g7a635cc-dirty

**Comment by judgeeeeee** [comment]
@simon-cj @FogDong

File: Makefile [hunk 1]
```diff
@@ -132,17 +144,53 @@ generate: controller-gen
 	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
 
 # Build the docker image
-docker-build: build-linux
-	@for target in $(TARGETS); do                                                      \
-	  image=$(IMAGE_PREFIX)$${target}$(IMAGE_SUFFIX);                                  \
-	  docker build -t $(REGISTRY)/$${image}:$(VERSION)                                 \
-	    --label $(DOCKER_LABELS)                                                       \
-	    -f $(BUILD_DIR)/$${target}/Dockerfile .;                                       \
+docker-build:
```
Comment (FogDong) [inline Makefile:147]: Maybe we shouldn't change the docker-build here but add a new command like "klever-publish-build"

File: Makefile [hunk 2]
```diff
@@ -132,17 +144,53 @@ generate: controller-gen
...
-	@for target in $(TARGETS); do                                                      \
-	  image=$(IMAGE_PREFIX)$${target}$(IMAGE_SUFFIX);                                  \
-	  docker build -t $(REGISTRY)/$${image}:$(VERSION)                                 \
-	    --label $(DOCKER_LABELS)                                                       \
-	    -f $(BUILD_DIR)/$${target}/Dockerfile .;                                       \
+docker-build: 
+	build
+	@for target in $(TARGETS); do  \
+		image=$(IMAGE_PREFIX)$${target}$(IMAGE_SUFFIX);   \
+		docker build -t $(REGISTRY)/$${image}:$(VERSION) --label $(DOCKER_LABELS)  -f $(BUILD_DIR)/$${target}/Dockerfile .;  \
```
Comment (FogDong) [inline Makefile:151]: Can we use ${RELEASE_VERSION} here instead of ${VERSION}?

File: Makefile (hunk 1 above)
Comment (FogDong) [reply Makefile:147]: Same for docker push

**Review by FogDong — COMMENTED** [review]
LGTM /cc @simon-cj

File: Makefile [hunk 3]
```diff
@@ -132,18 +144,85 @@ generate: controller-gen
...
+	done
+
+	#push serving 
+	@for target in $(SERVING_TARGETS); do  \
+		image=$(SERVING_IMAGE_PREFIX)$${target}$(SERVING_IMAGE_SUFFIX);   \
+		docker push  $(REGISTRY)/$${image}:$(VERSION);  \
+	done
+
+kelver-docker-build-push:
+	build
```
Comment (simon-cj) [inline Makefile:196]: This is a comment?

**Comment by simon-cj** [comment]
@judgeeeeee  记得加一下 description 和 release-note

**Pull request merged**